src/components/TransferPreview/TransferPreview.constants.ts

```diff
@@ -0,0 +1,21 @@
+const MESSAGES = {
+	TRANSFERRING_TO: 'is being transferred to:',
+	TRANSFER_IN_PROGRESS: 'Transferring...',
+	TRANSFER_TIME: 'This may take up to 20 minutes',
+	TRANSFER_TITLE: ' Transfer In Progress',
+};
+
+const URLS = {
+	ACCOUNT_ETHERSCAN: 'https://etherscan.io/address/',
+};
```

Comment:
Looks good to me. But we have `constatns/urls.ts` which has global URL constants. I think its good to move to there.